common/src/main/java/com/trainlab/model/AuthenticationInfo.java

```diff
@@ -3,15 +3,21 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import jakarta.persistence.Embeddable;
 import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 import lombok.Setter;
+import lombok.ToString;
 
 @Setter
 @Getter
 @AllArgsConstructor
 @NoArgsConstructor
 @Embeddable
+@Builder
+@ToString
+@EqualsAndHashCode
```

Comment:
```suggestion
@AllArgsConstructor
@NoArgsConstructor
@Embeddable
@Builder
@Data
```

Можно сократить несколько аннотаций